tests/unit/test_charm.py

```diff
@@ -344,13 +344,51 @@ def bundle_handler(args: ops.testing.ExecArgs) -> None:
     charm: DiscourseCharm = typing.cast(DiscourseCharm, harness.charm)
 
     email = "sample@email.com"
-    password = "somepassword"  # nosec
     event = MagicMock(spec=ActionEvent)
     event.params = {
         "email": email,
-        "password": password,
     }
-    charm._on_add_admin_user_action(event)
+    charm._on_promote_user_action(event)
```

Comment:
I cannot see the assert that checks that the rake command was called. I believe that something should be also checked/mocked with the rake task `users:exists`?